src/validation_workflow/validation_args.py

```diff
@@ -0,0 +1,74 @@
+# Copyright OpenSearch Contributors
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+import argparse
+import logging
+
+from validation_workflow.validation_rpm import ValidationRpm
+from validation_workflow.validation_tar import ValidationTar
+from validation_workflow.validation_yum import ValidationYum
+
+
+class ValidationArgs:
+    SUPPORTED_DISTRIBUTIONS = [
+        "tar",
+        "yum",
+        "rpm"
+    ]
+    SUPPORTED_PLATFORMS = ["linux"]
+    SUPPORTED_ARCHITECTURES = [
+        "x64",
+        "arm64",
+    ]
+    DISTRIBUTION_MAP = {
+        "tar": ValidationTar,
+        "yum": ValidationYum,
+        "rpm": ValidationRpm
+    }
```

Comment:
We should not need one list and one map just to rewrite all the dist again.

The design can be using 1 map max:

Example:

```

    DISTRIBUTION_MAP = {
        "tar": ValidationTar,
        "yum": ValidationYum,
        "rpm": ValidationRpm
    }
```

Then the platform/arch information can be distributed directly in the validation<> class.

Reply:
Sure Peter, I will use a single dictionary for storing distributions and mapping the distributions to respective classes. Could you please explain what `platform/arch information can be distributed directly in the validation<> class.`
in your comment means.Do you recommend that I declare these lists where they are actually implemented without declaring them in validationargs.py so I need not pass them as arguments to the function.